frontend/src/components/public_group/PublicGroupSignupV2.js

```diff
@@ -20,53 +21,102 @@ export default class PublicGroupSignupV2 extends React.Component {
+
+        <div className='max-width-2 mx-auto pt3 mt2'>
+          <div className="sm-flex items-stretch">
+            <div className="order-2 mb2">
+              <div className='pb2 pl2 pr2'>
+                <ImagePicker
+                  twitter={profileForm.attributes.twitterHandle || newUser.twitterHandle}
+                  email={profileForm.attributes.email || newUser.email}
+                  website={profileForm.attributes.website || newUser.website}
+                  src={newUser.avatar} // TODO pass user avatar, if they already have one. 
+                  className="avatar"
+                  handleChange={avatar => appendProfileForm({avatar})}
+                  {...this.props} // Pass uploadImage Action from `PublicGroup` container
+                />
+              </div>
+            </div>
+            <div className="sm-col-10 pr1 order-1 content-center">
+              <div className='pb20px'>
+                <label className='PublicGroup-font-15 pb1 -fw-normal display-none'>
+                  {i18n.getString('displayName')}
+                </label>
+                <Input
+                  type='text'
+                  placeholder='Your Name'
+                  value={profileForm.attributes.name || newUser.name}
+                  handleChange={name => appendProfileForm({name})}/>
+              </div>
+              <div className='pb20px'>
+                <label className='PublicGroup-font-15 pb1 -fw-normal display-none'>
+                  <span>{i18n.getString('website')}</span>
+                  <span className='pl1 h6 muted'>(Optional)</span>
+                </label>
+                <Input
+                  type='text'
+                  placeholder='Website URL (Optional)'
+                  value={profileForm.attributes.website || newUser.website}
+                  handleChange={
+                    (website) => {
+                      website = this.fixURI(website);
+                      appendProfileForm({website})
+                    }
+                  }/>
+              </div>
+              <div className='pb20px'>
+                <label className='PublicGroup-font-15 pb1 -fw-normal display-none'>
+                  <span>{i18n.getString('twitter')}</span>
+                  <span className='pl1 h6 muted'>(Optional)</span>
+                </label>
+                <Input
+                  type='text'
+                  placeholder='Twitter Handle (Optional)'
+                  value={profileForm.attributes.twitterHandle || newUser.twitterHandle}
+                  handleChange={twitterHandle => appendProfileForm({twitterHandle})}/>
+              </div>              
+            </div>
+
           </div>
           <div className="buttonsRow pt3">
             <SaveButton
               save={save.bind(this)}
-              label={i18n.getString('save')}
+              label="SHOW MY SUPPORT!" //  {/*i18n.getString('save')*/}
```

Comment:
My mistake, I changed the text to match the text in the design that cuiki gave me, I forgot to add it to the `strings.json` file. I'll add it now, unless you you want to keep the original string `Save` ?